osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -196,6 +198,8 @@ private void load()
                 IdleColour = colourProvider.Light4;
                 HoverColour = colourProvider.Light3;
                 blockedBackground.Colour = colourProvider.Background5;
+
+                drawableText.Text = text;
```

Comment:
Rather than hacking around the issue, wanna try and move the virtual call out of the constructor?

https://github.com/ppy/osu/blob/92f6f0207cf3310efdd91242a26fa6e2cbf2205f/osu.Game/Graphics/UserInterface/LoadingButton.cs#L49

Reply:
Sounds good. The loading spinner has to be instantiated in constructor though as there are properties that configures it (e.g. `LoadingAnimationSize`), I guess that's fine?

Reply:
Should be fine

Reply:
Had to move the `OnLoadStarted` and `OnLoadFinished` callbacks to `LoadComplete()` so that they don't get called while the content hasn't been initialised yet.

Correct way anyways as transforms shouldn't really happen before `LoadComplete`.